api/proto/teleport/legacy/types/webauthn/webauthn.proto

```diff
@@ -56,6 +56,36 @@ message SessionData {
   // "required".
   // An empty value is treated equivalently to "discouraged".
   string user_verification = 5 [(gogoproto.jsontag) = "userVerification,omitempty"];
+  // Scope authorized by this webauthn session.
+  ChallengeScope scope = 6 [(gogoproto.jsontag) = "scope,omitempty"];
```

Comment:
WDYT about moving scope and allow_reuse to a separate message, so we cleanly state that 1. This is not a webauthn feature, but instead a Teleport one and 2. All scope-related fields are grouped.

This is made even stronger by the fact that we see the same 2 fields repeated 3 times.